java-frontend/src/main/java/org/sonar/java/se/xproc/ExceptionalCheckBasedYield.java

```diff
@@ -24,23 +24,56 @@
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.commons.lang.builder.HashCodeBuilder;
 import org.sonar.java.se.ExplodedGraph.Node;
+import org.sonar.java.se.ProgramState;
 import org.sonar.java.se.checks.SECheck;
+import org.sonar.java.se.constraint.Constraint;
+import org.sonar.java.se.symbolicvalues.SymbolicValue;
 import org.sonar.plugins.java.api.semantic.Type;
 
 import java.util.Arrays;
+import java.util.List;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
 
 public class ExceptionalCheckBasedYield extends ExceptionalYield {
 
   private final Class<? extends SECheck> check;
+  private final MethodBehavior behavior;
 
   public ExceptionalCheckBasedYield(Type exceptionType, Class<? extends SECheck> check, Node node, MethodBehavior behavior) {
     super(node, behavior);
     this.check = check;
+    this.behavior = behavior;
```

Comment:
I don't see the point of storing the whole behavior in this case... varargs should be enough...